### PR TITLE
JAMES-2254 Introduce RPM packaging

### PR DESCRIPTION
**Pull request by rouazana**
This commits introduce RPM packaging.

### CODE REVIEW

File: dockerfiles/packaging/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM linagora/james-project:latest as source
+
+FROM debian:8.1
+
+RUN apt-get update
```
Comment (chibenwa) [inline dockerfiles/packaging/Dockerfile]: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers

 - Use && to minimize the number of layers
 - Clean your cache after this `rm -rf /var/lib/apt/lists/\*`

Comment (rouazana) [reply dockerfiles/packaging/Dockerfile]: what's the point as this image is only locally used?

Comment (chibenwa) [reply dockerfiles/packaging/Dockerfile]: It takes less time to generate it... Anyway, it's not very important IMO

**Comment by chibenwa** [comment]
Merged

File: README.adoc [hunk 2]
```diff
@@ -351,23 +351,25 @@ Where :
 - SHA1: is the given git SHA1 of the james-project repository to merge.
 - RESULTING_BRANCH: is the branch created when merging.
 
-=== Generating a debian package for James
+=== Generating a package for James
 
-You can generate a debian package for James by using the following process.
+You can generate a Debian package or an RPM package for James by using the following process.
```
Comment (mbaechler) [inline README.adoc]: s#RPM#REHL/CentOS#

Comment (rouazana) [reply README.adoc]: do you think it won't work on Suse? :)

Comment (rouazana) [reply README.adoc]: or Mandriva :trollface:

Comment (mbaechler) [reply README.adoc]: Either you write "deb / rpm" or distribution names, but the mix is unfortunate

Comment (rouazana) [reply README.adoc]: changed

**Pull request merged**